### DISCUSSION TITLE
can we add an example with electron / TauriJS ?

### DISCUSSION THREAD

**Discussion by ddublon** [noam-honig/radweb#443] [Q&A]
HI there,

I really want to use remult

but my front-end is also built with TauriJS

is there any way to use remult with TauriJS?

do you have an idea how to do it?

**Comment by noam-honig** [comment]
Remult works great with any javascript platform - including electron / TauriJS.

Did you run into any specific challenges?

**Reply by ddublon** [reply]
yes, TauriJS is already a Rust server, that serves the front end as a static website, so what can I do to use Remult in between? I don't understand

**Reply by ddublon** [reply]
do you have time today ? for an e-meeting ?

**Reply by noam-honig** [reply]
Ping me on discord I'll send you a meeting link
